packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -109,14 +109,9 @@ export namespace CommonCommands {
         id: 'core.redo',
         label: 'Redo'
     };
-
-    export const FIND: Command = {
```

Comment:
We should it in CHANGELOG and specify what should be called instead.

Reply:
👍 Yeah, I have to update the changelog.

Reply:
Done.